### ISSUE TITLE
Graceful Edge upgrade over the air (OTA)

### ISSUE THREAD

**Issue by ivelin** [ambianic/ambianic-edge#166]
- [ ] Remote update when new version is available
- [ ] Three update profiles - recommended (user doesn't need to see and can ignore), urgent (user must be notified), critical (user must take action: agree or reject).
- [ ] Manual vs Automated update option. User can choose to enable automated updates.

**Comment by ivelin** [comment]
What are the stable options for Raspberry Pi & docker OTA updates?

Resources:
https://hub.mender.io/t/how-to-update-a-docker-container-with-a-raspberry-pi-3/1106
https://www.balena.io/docs/reference/OS/updates/update-process/

**Comment by ivelin** [comment]
We introduced [docker-compose.yaml with watchtower](https://gist.github.com/ivelin/3891a7b5d61a12d6a1b9f652b6d53dce) in our [quick start guide](https://docs.ambianic.ai/users/quickstart/) about a month ago. It has worked well for users and has been able to seamlessly update ambianic-edge when new versions are released.

Closing this issue as the watchtower solution has worked well and it is no longer a pain point reported by end users.
